### ISSUE TITLE
Can we publish a public image on dockerhub or other repo?

### ISSUE THREAD

**Issue by jxlwqq** [kubewharf/godel-scheduler#48]

**Comment by NickrenREN** [comment]
any plans for this ? @zryfish

**Comment by NickrenREN** [comment]
We plan to release v0.1 this quarter, maybe we can build a public image based on that.

**Comment by zryfish** [comment]
> We plan to release v0.1 this quarter, maybe we can build a public image based on that.

Yes, I agree with that. It's better to build a public image based on a released version.
